.travis.yml

```diff
@@ -1,32 +0,0 @@
-dist: focal
-language: python
-install:
-  - pip install tox-travis
-  - pip install codecov
-matrix:
-  include:
-    - python: 3.6
-      env: ENV=pep8,py3
-    - python: 3.7
-      env: ENV=pep8,py3
-    - python: 3.8
-      env: ENV=pep8,py3
-    - env: ENV=func
```

Comment:
Don't we run this anymore? These tests were making use of juju, see below

Reply:
Yes, but right now the tests are disabled and we have lint errors in master as a consequence, I'd be happy to re-add the functional tests (I added the original one) another day

Reply:
Why another day? Are they failing? Maybe if you re-enable them now they'll just be green? Or does it involve more work to give it a try?

Reply:
Don't feel blocked, but I think it's worth a try and if it's red I'm happy to give up for now

Reply:
Moving it over from the non-running and non-functional Travis recipe to the GitHub Actions recipe is a bit more involved than copy pasta, and I'm out of runway for today. I'm in no rush to land this and can circle back to it when convenient later in the week, no worries. In the mean time lets leave it open in case anyone needs it to work around current juju/lxd issues when deploying on MAAS re [bug 1942864](https://bugs.launchpad.net/juju/+bug/1942864)

Reply:
OK I didn't realize this was broken and involved more than copy-pasting a few lines. Let's land this for now then, thanks!